chirpus-two/index.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans&display=swap');
+
+body {
+    margin:10px;
+    background-color:#f9fafc ;
+    font-family: 'Open-Sans', sans-serif;
+}
+
+.wrapper {
+    display: flex;
+    flex-direction: column;
+}
+
+a {
+    text-decoration: none;
+    color: black;
+    padding-left: 5px;
+
+}
+
+.nav {
+    display: flex;
+    flex-direction: column;
+}
+.nav__link {
+    color:#69696a;
+    padding: 5px;
+}
+h5 {
+    margin: 0;
+    color:black;
+    padding: 5px;
+}
+
+.content p {
+    border-left: 2px solid blue;
+    padding: 0 5px;
+    font-size: 14px;
+    color:#69696a;
+    border-left: 3px solid #69696a;
+}
+
+
+
+.button {
+    background-color: black;
+    border: none;
+    border-radius: 10px;
+    color: white;
+    padding: 5px;
+    width: 100%;
+}
+.graphic {
+   display: none;
+   position: relative;
+}
+
+.wrapper {
+    display: flex;
+    flex-direction: row;
+}
+ 
+
+@media (min-width: 768px) {
+    .nav{
+        flex-direction: row;
+        align-items: center;
+        justify-content: flex-end;
+    }
+    .nav h5 {
+        margin-right: auto;
```

Comment:
This is the only build spec that you missed. Spec asks "A flex property is used to separate “chirpus inc” and the navigation items", but you separated them with margin-right.